index.js

```diff
@@ -0,0 +1,27 @@
+// const fs = require('fs');
+
+// const sumNumbers = (fileA, fileB) => {
+//     const numberA = parseInt(fs.readFileSync(fileA, 'utf8'));
+//     const numberB = parseInt(fs.readFileSync(fileB, 'utf8'));
+//     const sum = numberA + numberB;
+//     console.log(sum);
+// }
+
+const fs = require('fs/promises');
+
+const sumNumbers = async (fileA, fileB) => {
```

Comment:
It would be better to rename this function or create separate functions to parse numbers from files and calculate their sum